Sources/Raft/Consensus/Consensus.swift

```diff
@@ -0,0 +1,227 @@
+// SPDX-License-Identifier: MIT
+// Copyright © 2021 makadaw
+
+
+import struct Dispatch.DispatchTime
+import enum Dispatch.DispatchTimeInterval
+import Logging
+
+actor public class Consensus {
```

Comment:
probably 
```suggestion
public actor class Consensus {
```

as a heads up, this soon will be just `public actor`

Reply:
Yes, with last snapshot (16-02) just `public actor` works